test/extensions/filters/common/rbac/matchers_test.cc

```diff
@@ -233,6 +243,38 @@ TEST(PortMatcher, PortMatcher) {
   checkMatcher(PortMatcher(456), false, conn, headers, info);
 }
 
+TEST(PortRangeMatcher, PortRangeMatcher) {
```

Comment:
Please add a short comment stating what the test validates.

Reply:
Ok, comment added.